lib/charms/operator_libs_linux/v0/juju_systemd_notices.py

```diff
@@ -386,7 +422,7 @@ async def _async_load_services() -> None:
 
     # Loop through all the services and be sure that a new watcher is
     # started for new ones.
-    for service in watched_services:
+    for service in watched_services.keys():
```

Comment:
Why is `.keys()` necessary here? Iterating keys is the default for dicts.

Reply:
Force of habit :sweat_smile: